pkg/server/container_create.go

```diff
@@ -321,6 +322,8 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 	// Set the relative path to the rootfs of the container from containerd's
 	// pre-defined directory.
 	g.SetRootPath(relativeRootfsPath)
+	// add default UNIX path; overwritten optionally by image config PATH env
+	g.AddProcessEnv("PATH", "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin")
```

Comment:
Do this in `defaultRuntimeSpec`.

Reply:
Done; added to section where `spec.Linux` is non-nil as it only makes sense to have a default UNIX-like `PATH` for Linux specs.

Reply:
We only support linux in 1.2 actually. :)

But this is fine.